hack/.golint_failures

```diff
@@ -1,4 +1,5 @@
 cluster/images/etcd-version-monitor
+cmd/controller-manager/app/options
```

Comment:
can you fix the golint errors instead of adding it here?

Reply:
fixed now